packages/fxa-content-server/app/scripts/models/account.js

```diff
@@ -644,7 +645,7 @@ const Account = Backbone.Model.extend(
           // originally used for login.
           if (
             options.originalLoginEmail &&
-            email.toLowerCase() !== options.originalLoginEmail.toLowerCase()
+            emailsMatch(email, options.originalLoginEmail)
```

Comment:
And here :-)

Reply:
Yiiiiiikes. Guess the repetitiveness got to me. Thanks for this!